.github/workflows/build-push.yml

```
@@ -0,0 +1,103 @@
+name: Build-Push
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Could you bump this to v3:
```suggestion
    - uses: actions/checkout@v3
```